README.md

```diff
@@ -18,7 +18,8 @@ This action accepts the following inputs:
 - `oxygen_deployment_token`: The Oxygen deployment token issued by Shopify. This is a required input.
 - `oxygen_worker_dir`: The name of the directory containing the worker file. Defaults to `dist/worker`.
 - `oxygen_client_dir`: The name of the directory with compiled asset files to be uploaded to the CDN. Defaults to `dist/client`.
-- `oxygen_health_check`: Determines whether to ensure the preview url is accessible on Oxygen before marking the workflow as successful. This does **not** mean the deployment wasn't successful, but that the page may not be loading properly. The deployment status in the Hydrogen Storefronts channel will reflect the status of the deployment, not the health of the deployment. Accepts `true` or `false`. Defaults to `true`.
+- `oxygen_verify_deployment`: Verifies succesful deployment of the worker to Oxygen. Accepts `true` or `false`. Defaults to `true`. Note: This verification checks the status of the deployment, not the health of the application. If set to true, the workflow will only be marked as successful if the worker deployment to Oxygen has been completed successfully.
+- `oxygen_health_check` (deprecated): Previously used to ensure the application is reachable on Oxygen before marking the deployment as successful. This input is now deprecated and `oxygen_verify_deployment`` should be used instead.
```

Comment:
Since the default it `true` we will just turn it on if they've specified `false` here. I don't know if I see that as a huge problem as we'll never be able to remove this otherwise unless we release a new version of the action. I say let's just 🔥 it, IMO not super impactful and IIRC github will just warn you that you're using an unsupported input or something so they can investigate and change it.